apps/designer/components/Buttons/HandleToggle.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const HandleToggle = ({
+  toggleButton,
+  myToggleButton,
+}: {
+  toggleButton: boolean;
```

Comment:
Instead of this:

```tsx
const HandleToggle = ({
   toggleButton,
   myToggleButton,
 }: {
   toggleButton: boolean;
myToggleButton: CallableFunction;
 }) => { 
  return(
    ...
)
}
```

Prefer this:
```tsx
interface ToggleButtonProps {
  isToggled: boolean;
  handleClick: CallableFunction;
}

class ToggleButton = ({ props }: ToggleButtonProps) => ( ... )
```